### ISSUE TITLE
Jet Widget

### ISSUE THREAD

**Issue by hershg21** [FrSkyRC/ETHOS-Feedback-Community#4917]
after down load new1.6.0 firmware I get no sense of setup with LUI scripts for my SWIWIN turbine. Have been using another brand LUI with ease of setup.

**Comment by Bobbysonic** [comment]
I agree, there is no instruction anywhere to help set-up this widget! Also, make sure you use the new widget screen at the very bottom.  Please provide notes to this new feature FrSky...

**Comment by robthomson** [comment]
What widget or lua are you even talking about?

Sorry.. But this request has been written with the assumption that the end reader knows what you have just been looking at.

**Comment by Bobbysonic** [comment]
I apology for not being more clear! I`m referring to the new Jet Dashboard widget on 1.6.0 update.  Sitting up the widget using the Swinwin turbine telemetry, The parameters requirements not understood how they apply with sensors discovered or where to find them.  Example; Speed Configure, Speed source, Fuel source etc. Unable to find information to input.

**Comment by hershg21** [comment]
You get your.
Turbine information. Once you power up the turbine itself, do a telemetry
sensor discovery, it will populate the information from the turbut where
I'm confused is. Where do I apply that information to the widget?

**Comment by bsongis-frsky** [comment]
This widget is just an example, you may adapt it to fit to your needs, or request for help, rather in the Discord server, or RCG, or FB
